nowcasting_dataset/data_sources/gsp/gsp_data_source.py

```diff
@@ -225,8 +221,14 @@ def get_example(
             dims=["time", "id"],
         )
 
+        capacity = xr.DataArray(
+            data=selected_capacity.values,
+            dims=["time", "id"],
+        )
+
         # convert to dataset
         gsp = da.to_dataset(name="data")
+        gsp["capacity_mwh"] = capacity
```

Comment:
MWh?  Or `capacity_mw`?  :slightly_smiling_face:   (sorry if I've misunderstood!)

Reply:
sorry im too much int he battery world - hmmm, is capacity_mw ok? It seems wrong to me, but [google](https://www.google.com/search?q=pv+capacity&rlz=1C5CHFA_enGB966GB966&oq=pv+capacity&aqs=chrome..69i57j0i512l6j0i22i30l3.4862j0j7&sourceid=chrome&ie=UTF-8) seems to say that 'capacity_mw' is ok

Reply:
Ah, yes, I see!

I see the confusion:  "battery capacity" is MWh :slightly_smiling_face: 

I'm pretty sure "PV capacity" is measured in terms of the peak _power_ output (MW); rather than _energy_ (MWh).

Reply:
I maybe use 'mwp' - mega watts peak
This is used by [pvlive](https://github.com/SheffieldSolar/PV_Live-API) and [google](https://www.google.com/search?q=units+mwp+power&rlz=1C5CHFA_enGB966GB966&ei=ETmVYfbiOIfFgQaZq6uIBA&oq=units+mwp+power&gs_lcp=Cgdnd3Mtd2l6EAMyCAghEBYQHRAeOgcIABBHELADOgYIABAWEB5KBAhBGABQsgNYoAhgigloAXABeACAAXyIAYMEkgEDMi4zmAEAoAEByAEIwAEB&sclient=gws-wiz&ved=0ahUKEwi2r8ew85_0AhWHYsAKHZnVCkEQ4dUDCA4&uact=5)